src/SonarQube.Client/Models/Connections.cs

```diff
@@ -58,13 +78,7 @@ public AuthenticationType Authentication
             get;
         }
 
-        internal /*for testing purposes*/ bool IsDisposed
-        {
-            get
-            {
-                return this.isDisposed;
-            }
-        }
+        public bool IsDisposed => this.isDisposed;
```

Comment:
Is this _public_ for testing? The other SonarLint projects use _InternalsVisibleTo_